gui/wxpython/gui_core/widgets.py

```diff
@@ -435,7 +435,7 @@ class NumTextCtrl(TextCtrl):
     """Class derived from wx.TextCtrl for numerical values only"""
 
     def __init__(self, parent, **kwargs):
-        ##        self.precision = kwargs.pop('prec')
+        #        self.precision = kwargs.pop('prec')
```

Comment:
This is very old and not relevant, please just remove it completely

Reply:
Got it, I have removed the whole comment

Reply:
Got it, I have removed the whole comment